testing/game/Game001 - start game.md

```diff
@@ -4,27 +4,32 @@
 >this testcase verifies that the team can start the game correctly
 
 *Preconditions*
+* Language is set to English
```

Comment:
Hm, dotaz. Chceme ty testy mít pro angličtinu, i když je to jakoby druhořadý jazyk pro luštěniny?

Reply:
Hm, dobrá otázka... Asi to spíš uděláme v češtině...